src/actionlib/index.js

```diff
@@ -5,7 +5,6 @@ var action = module.exports = {
     ActionClient: require('./ActionClient'),
     ActionListener: require('./ActionListener'),
     Goal: require('./Goal'),
-    SimpleActionServer: require('./SimpleActionServer')
```

Comment:
shouldn't this stay if we're keeping this for ros1 compat?

Reply:
oof yeah, missed this revert. Nice catch